### ISSUE TITLE
Scaling should not be ignored part deux

### ISSUE THREAD

**Issue by Myndex** [xi/apca-introduction#14]
New issue as you closed the other without resolution.

I have been trying to explain to you why you are wrong in a way that you can understand. Since you are a math major, try this:

How do you determine the hypotenuse (c)? Euclid instructs us:

### a² + b² = c²

But you are then claiming that a + b = c, and this is not true.

The square root of (a² + b²) IS NOT equal to (a + b), yet this is what you are claiming with your monotonic argument.

But in the "analysis" you are doing, you are essentially making that assertion.

You are also comparing "non-perceptual lightness" of WCAG 2's luminance to the _perceptual lightness curves of APCA_. This is a fully invalid comparison. It is an "apples and apricots" comparison at best.

Your assertion that 0.4 is an ambient flare component is fully unsupported by any science. A trivial measurement of the actual flare in anything resembling a standardized environment demonstrates that this is far from the case, and has absolutely no basis in fact.

## An Alternate Examples

Going back over the 2019 trials, I revisited an earlier model, and I just released it at [DeltaPhiStar](https://github.com/Myndex/deltaphistar)  as a general purpose perceptual contrast algorithm.

```js

   deltaPhiStar = (Math.abs(bgLstar ** 1.618 - txLstar ** 1.618) ** 0.618) * 1.414 - 40 ;
    
    // ** is equiv to Math.pow
``` 

You could remove the final scaling so that:

$$
(bgLstar^{1.618} - txLstar^{1.618})^{0.618}
$$

But this is irreducible. You can not:

$$
( bgLstar - txLstar )
$$

And hope to be in any way similar in result.

In short, you are stating that you are only "examining the mathematical properties" while ignoring the "visual science aspects" which is a spurious and incongruent argument when the math is specifically modeling the visual quantities.

**Comment by Myndex** [comment]
THis is a continuation of #7 which was closed

**Comment by xi** [comment]
Duplicate of #7. Please comment there if you have new arguments. I will see it and reopen if new arguments are brought to the table (which I do not see here).
